crates/sqlbuiltins/src/functions/mod.rs

```diff
@@ -19,6 +19,7 @@ use table::{BuiltinTableFuncs, TableFunc};
 
 /// Builtin table returning functions available for all sessions.
 static BUILTIN_TABLE_FUNCS: Lazy<BuiltinTableFuncs> = Lazy::new(BuiltinTableFuncs::new);
+// TODO: Why is this separate?
```

Comment:
The arrow cast function has special handling in datafusion due to the dynamic return type. So it isn't a 'scalar_udf' and needs to be handled a bit differently.

Reply:
This constant wasn't used, so removed it. Also added a comment when generating the hashmap about the arrow_cast special case.